src/Uno.UI.Toolkit/Diagnostics/DiagnosticsOverlay.cs

```diff
@@ -381,7 +381,15 @@ protected override void OnApplyTemplate()
 			//_anchor.Tapped += OnAnchorTapped;
 			_anchor.ManipulationDelta += OnAnchorManipulated;
 			_anchor.ManipulationCompleted += OnAnchorManipulatedCompleted;
-			_anchor.ManipulationMode = ManipulationModes.TranslateX | ManipulationModes.TranslateY | ManipulationModes.TranslateInertia;
+
+			_anchor.ManipulationMode = ManipulationModes.TranslateX | ManipulationModes.TranslateY;
+#if HAS_UNO
+			if (ManipulationModesExtensions.IsSupported(ManipulationModes.TranslateInertia))
```

Comment:
`TranslateInertia` should be supported on all platforms. If this is the cause of the issue for the HR indicator, then we should fix the `GestureRecognizer` not disable the inertia.

Reply:
BTW @Kunal22shah I'm looking to fix the issue with the inertia processor